OOP2/MainOOP2.java

```diff
@@ -0,0 +1,82 @@
+package OOP;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class MainOOP2 {
+    public static void main(String[] args) {
+
+        Scanner reader= new Scanner(System.in);
+
+        //Numbers
+        ArrayList listOfNumbers = new ArrayList<>();
+        listOfNumbers.add(55);
+        listOfNumbers.add(5);
+        listOfNumbers.add(7);
+        listOfNumbers.add(100);
+
+        Numbers numbersToConstructor = new Numbers(listOfNumbers);
+        System.out.println("The numbers from the array are: " + listOfNumbers);
```

Comment:
Even though this will give you elements, usually, to show elements, we need to iterate through ArrayList and print them